news/2017-11-05-Official-Mapping-Contests:-osu!-Mapping-Olympiad.md

```diff
@@ -0,0 +1,71 @@
+---
+layout: post 
+title: "Official Mapping Contests: osu! Mapping Olympiad"
```

Comment:
A title shouldn't read like you're naming a folder or something, currently that reads like `mapping contests/olympiad` like you're building up some weird folder structure.
All the other stuff has at least the order reversed where it's reading like: osu! Mapping Olympiad! Sign up, win stuff! or something among those lines, basically main topic first instead of general categorizing. Besides anything on the news can be treated as `officially approved of or tolerated` anyways. Adding that it's an official contest adds no information for anyone reading it.

Reply:
I kind of disagree with what you've said, I've taken an example from the current newsposts and they seem to have the same structure. Do you have a better idea?